ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ViewVulnReportPage.tsx

```diff
@@ -218,23 +219,42 @@ function ViewVulnReportPage() {
                     </Tab>
                     <Tab
                         eventKey={1}
-                        title={<TabTitle icon={<HistoryIcon />}>Run history</TabTitle>}
-                        aria-label="Run history tab"
+                        title={<TabTitle icon={<ClipboardCheckIcon />}>All report jobs</TabTitle>}
+                        aria-label="Report jobs tab"
+                        actions={
+                            <>
+                                <Popover
+                                    aria-label="All report jobs help text"
+                                    headerContent={<div>All report jobs</div>}
+                                    bodyContent={
+                                        <div>
+                                            This function displays the requested jobs from different
+                                            users and includes their statuses accordingly. While the
+                                            function provides the ability to monitor and audit your
+                                            active and past requested jobs, we suggest configuring
+                                            the &quot;Vulnerability report retention limit&quot;
```

Comment:
Just an idea as a follow up, it might be useful to link to the "System Configuration" page here.